tests/test_envs.py

```diff
@@ -13,6 +15,8 @@
 from minigrid.core.mission import MissionSpace
 from tests.utils import all_testing_env_specs, assert_equals
 
+os.environ["SDL_VIDEODRIVER"] = "dummy"
```

Comment:
Replace with the sdl videodriver with https://github.com/Farama-Foundation/Gymnasium/blob/main/gymnasium/__init__.py

Reply:
remove for the other two cases

Reply:
The test_scripts file has a test_manual_control test which uses human rendering to check that the env's `close()` calls pygame.quit and that the manual controls are working properly, so I kept it there. Test wrappers doesn't need it though so I removed it there.